dis_snek/api/http/http_requests/invites.py

```diff
@@ -0,0 +1,20 @@
+from typing import Any
+
+from dis_snek.client.const import MISSING, Absent
+from ..route import Route
+
+
+class InviteRequests:
+    request: Any
+
+    async def delete_invite(self, invite_code: str, reason: Absent[str] = MISSING) -> dict:
```

Comment:
This is a duplicate of a function we already have in https://github.com/Discord-Snake-Pit/Dis-Snek/blob/e85c04abf42650958a2a750674650cccbc6456f1/dis_snek/api/http/http_requests/channels.py#L241
I do not think this file is needed.